### ISSUE TITLE
Add a License

### ISSUE THREAD

**Issue by logan-gatlin** [felipec/vim-felipec#2]
Awesome color pallet! I wanted to use it on my website for my code blocks, but noticed there is no license. It would be great if you could clarify how and where your work can be used. I noticed that the [highlight.js](https://highlightjs.org/) uses your pallet and credits you, but technically they don't have the right because you haven't added a license. If this is intentional and you don't want your pallet used in other projects, you should probably go bother them because their project seems quite popular. Thanks!

**Comment by felipec** [comment]
Technically as the copyright holder I don't need to add a license, I can just pick and choose who I allow to use it, and given that it was me the one that sent the patch to highlight.js, it's a given that I'm allowing them to use it and under their license.

That being said, I've added a [LICENSE](https://github.com/felipec/vim-felipec/blob/master/LICENSE) file to make it explicit how you can use this project without my approval.

Basically I'm fine with anybody using this palette, but if attribution is possible, it should be attributed.
